cli/rename.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// RenameCommand use to implement 'rename' command, it rename a container.
+type RenameCommand struct {
+	baseCommand
+}
+
+// Init initialize rename command.
+func (rc *RenameCommand) Init(c *Cli) {
+	rc.cli = c
+
+	rc.cmd = &cobra.Command{
+		Use:   "rename [container] [newName]",
+		Short: "Rename a exist container with newName",
+		Args:  cobra.MinimumNArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return rc.runRename(args)
+		},
+	}
+}
+
+// runRename is the entry of rename command.
+func (rc *RenameCommand) runRename(args []string) error {
+	apiClient := rc.cli.Client()
+	container := args[0]
+	newName := args[1]
+
+	if err := apiClient.ContainerRename(container, newName); err != nil {
+		fmt.Fprintf(os.Stderr, "failed to rename container %s to %s %v \n", container, newName, err)
```

Comment:
Please do not output error here, you can just `return err`, since https://github.com/alibaba/pouch/blob/master/cli/main.go#L31 will output error message.